src/shared/Microsoft.Git.CredentialManager/GitConfiguration.cs

```diff
@@ -3,117 +3,121 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Linq;
 using System.Text;
 
 namespace Microsoft.Git.CredentialManager
 {
     /// <summary>
     /// Invoked for each Git configuration entry during an enumeration (<see cref="IGitConfiguration.Enumerate"/>).
     /// </summary>
-    /// <param name="name">Name of the current configuration entry.</param>
-    /// <param name="value">Value of the current configuration entry.</param>
+    /// <param name="entry">Current configuration entry.</param>
     /// <returns>True to continue enumeration, false to stop enumeration.</returns>
-    public delegate bool GitConfigurationEnumerationCallback(string name, string value);
+    public delegate bool GitConfigurationEnumerationCallback(GitConfigurationEntry entry);
 
     public enum GitConfigurationLevel
     {
         All,
-        ProgramData,
```

Comment:
Whoa, that was still in there, wasn't it.

Reply:
Yep, but was totally unused in the GCM code anyway.. along with the `Xdg` member. Hence consolidating them to `Unknown` (again, just a name for values we don't care about).